src/app.js

```diff
@@ -1,21 +1,53 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const url = require('url');
+
+const PORT = process.env.PORT || 3000;
+// eslint-disable-next-line max-len
+const WRONG_PATH_MESSAGE = 'No such file or path to file is invalid. Path shoud start with "/file" and relative path.';
+
+const server = http.createServer((req, res) => {
+  let filePath = req.url;
+
+  if (filePath.endsWith('/')) {
+    filePath = filePath.slice(0, -1);
+  }
+
+  if (filePath === '/file') {
+    filePath += '/index.html';
+  }
+
+  try {
+    if (!filePath.startsWith('/file')) {
+      throw new Error(WRONG_PATH_MESSAGE);
+    }
+
+    const fullPath = path
+      .join(__dirname, '..', filePath.replace('file', 'public'));
```

Comment:
Don't join __dirname here. Dirname is `src`, public folder should not be there, it should be on the same level as `src`